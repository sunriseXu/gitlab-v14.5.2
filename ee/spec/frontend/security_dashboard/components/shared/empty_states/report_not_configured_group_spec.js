import { shallowMount } from '@vue/test-utils';
import ReportNotConfiguredGroup from 'ee/security_dashboard/components/shared/empty_states/report_not_configured_group.vue';

describe('Group report not configured component', () => {
  let wrapper;
  const dashboardDocumentation = '/path/to/dashboard/documentation';
  const emptyStateSvgPath = '/placeholder.svg';

  const createWrapper = () =>
    shallowMount(ReportNotConfiguredGroup, {
      provide: {
        dashboardDocumentation,
        emptyStateSvgPath,
      },
    });

  beforeEach(() => {
    wrapper = createWrapper();
  });

  afterEach(() => {
    wrapper.destroy();
  });

  it('matches snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
