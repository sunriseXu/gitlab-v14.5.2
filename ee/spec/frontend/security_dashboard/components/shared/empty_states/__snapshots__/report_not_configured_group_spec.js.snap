// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group report not configured component matches snapshot 1`] = `"<gl-empty-state-stub title=\\"Monitor vulnerabilities in your group\\" svgpath=\\"/placeholder.svg\\" description=\\"Manage and track vulnerabilities identified in projects within your group. Vulnerabilities in projects are shown here when security testing is configured.\\" secondarybuttonlink=\\"/path/to/dashboard/documentation\\" secondarybuttontext=\\"Learn more\\"></gl-empty-state-stub>"`;
